scripts/deploy.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+
+# Function to check if a command is available
+command_exists() {
+  command -v "$1" >/dev/null 2>&1
+}
+
+# Check if jq is installed
+if ! command_exists jq; then
+  echo "jq is not installed. Installing..."
+  sudo apt-get update
+  sudo apt-get install -y jq
+fi
+
+# Check if sed is installed
+if ! command_exists sed; then
+  echo "sed is not installed. Installing..."
+  sudo apt-get update
+  sudo apt-get install -y sed
+fi
```

Comment:
Woha. This is totally out of scope for a script like this.

Have you considered using JS for a script? ([`/module/include.js`](https://github.com/near/neardevhub-widgets/blob/main/module/include.js) historically managed the replacements)

Reply:
@elliotBraem You may take inspiration from my demo TypeScript starter where I also generate a build folder on the fly: https://github.com/frol/bos-component-ts-starter/blob/main/package.json#L25 (see `node ./build.js` at the end, and see the [`build.js` file](https://github.com/frol/bos-component-ts-starter/blob/main/build.js))

Reply:
I've removed the deploy script from all of the plugins. I think this would make a great ticket